examples/prerecorded_from_url.rs

```diff
@@ -20,17 +20,37 @@ async fn main() -> Result<(), DeepgramError> {
     let source = AudioSource::from_url(AUDIO_URL);
 
     let options = Options::builder()
```

Comment:
Enabled all features except callback and callback_method to show call signatures